src/components/Footer.tsx

```diff
@@ -0,0 +1,89 @@
+import { Dispatch, FC } from 'react';
+
+import { deleteTodo } from '../api/todos';
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  selectedStatus: Status;
+  setSelectedStatus: (status: Status) => void;
+  setTodos: Dispatch<React.SetStateAction<Todo[]>>;
+  setIsDeleting: Dispatch<React.SetStateAction<number>>;
```

Comment:
This property is not clear/redundant
Please remove or rename and change the type (without react.Dispatch)